apps/debug/client/uos/commos.c

```diff
@@ -77,10 +105,87 @@ DbgrConsoleInterruptHandler (
 //
 
 //
+// List of supported baud rates (taken from apps/swiss/stty.c)
```

Comment:
@evangreen maybe it makes sense to share this table with `swiss/stty`? After all both utililties live in userspace.